doc/translatable.md

```diff
@@ -3,7 +3,9 @@
 **Translatable** behavior offers a very handy solution for translating specific record fields
 in different languages. Further more, it loads the translations automatically for a locale
 currently used, which can be set to **Translatable Listener** on it`s initialization or later
-for other cases through the **Entity** itself
+for other cases through the **Entity** itself.
+
+Please note that **Translatable** behaviour is meant to be used for small projects and prototypes because of performance limitations.
 
```

Comment:
**NOTE:** that **Translatable** uses string type as foreignKey in order to be compatible with more types of columns.
When your database grows over 10K translated fields, if your **foreignKey** actually an **integer** for better performance update this column type, see #1512 for more details. If the performance gains are not well enough, consider a custom implementation for your translations.